pkg/tabulator/tabstate.go

```diff
@@ -198,8 +217,7 @@ func Update(ctx context.Context, client gcs.ConditionalClient, mets *Metrics, co
 				}
 			}
 			if filter {
-				groupCfg := cfg.Groups[tab.GetTestGroupName()]
-				err := createTabState(ctx, log, client, tab, groupCfg, *fromPath, *toPath, confirm, dropEmptyCols, calculateStats, useTabAlertSettings)
+				err := createTabState(ctx, log, client, view.tab, group, *fromPath, *toPath, confirm, dropEmptyCols, calculateStats, useTabAlertSettings)
```

Comment:
Seems like we should refactor group out of this function.

Instead of downloading and inflating the group state n-view times we can do this once, and then send a copy of the inflated state to all the views for further processing.

(but we'll have to be careful to do a deep copy or whatnot as maps and slices are passed as references)